code/game/machinery/computer/communications.dm

```diff
@@ -111,9 +111,10 @@
 					if(GLOB.security_level != old_level)
 						to_chat(usr, "<span class='notice'>Authorization confirmed. Modifying security level.</span>")
 						playsound(src, 'sound/machines/terminal_prompt_confirm.ogg', 50, 0)
-						//Only notify the admins if an actual change happened
+						//Only notify people if an actual change happened
 						log_game("[key_name(usr)] has changed the security level to [get_security_level()].")

```

Comment:
get_security_level should probably be cached to a var instead of being called 3 times repeatedly.